src/components/media/audio/sound.tsx

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable no-magic-numbers */
+import './sound.css'
+
+import ClosedCaptionDisabledRoundedIcon from '@mui/icons-material/ClosedCaptionDisabledRounded'
+import ClosedCaptionRoundedIcon from '@mui/icons-material/ClosedCaptionRounded'
+import { useMediaQuery, useTheme } from '@mui/material'
+import Alert from '@mui/material/Alert'
+import CircularProgress from '@mui/material/CircularProgress'
+import IconButton from '@mui/material/IconButton'
+import Typography from '@mui/material/Typography'
+import { Box } from '@mui/system'
+import React, { useEffect, useRef, useState } from 'react'
+
+import MoveTenSecondsButton from '../moveTenSecondsButton/moveTenSecondsButton'
+import PausePlayToggle from '../pausePlayToggle/pausePlayToggle'
+import PlaybackRateButton from '../playbackRateButton/playbackRateButton'
+import ProgressSlider, {
+  type BufferRange,
+} from '../progressSlider/progressSlider'
+import TimeIndicator from '../timeIndicator/timeIndicator'
+import Transcript from '../transcript/transcript'
+import TranscriptToggle from '../transcriptToggle/transcriptToggle'
+import VolumeSettings from '../volumeSettings/volumeSettings'
+
+export interface SoundProps {
```

Comment:
I think you need to update the type to be consistent with the data format change in the previous PR.